docs/projects-to-work-on.md

```diff
@@ -0,0 +1,428 @@
+shine when it comes do diverse and ever-present documentation. If you would like
+to help fix this, here are a few ideas.
+
+For beginners and new Ada programmers:
+
+- **[Rosetta Code](https://rosettacode.org/wiki/Rosetta_Code):** Rosetta
+  provides examples and solutions to common problems and tasks in many
+  programming languages. It serves as a learning resource as well as a
+  comparison tool between languages. Ada already has a lot of examples, but
+  still, some are missing compared to other languages. And some of the examples
+  are quite old and they could be updated and cleaned up. These tasks should be
+  fairly easy to carry out and they will help you learn Ada along the way!
+- **[GeeksforGeeks](https://www.geeksforgeeks.org/)** and **[Tutorials
+  Point](https://www.tutorialspoint.com/):** these well-known websites have
+  resources on different languages and topics about them, ranging between simple
+  syntax elements to solving specific tasks in them. Ada has little to no
+  presence in them. It would be nice if we could document Ada in them or point
+  people to use [Learn Ada](https://learn.adacore.com/) for more information.
+
+For intermediate and seasoned Ada and SPARK users:
+
+- **[Alice](https://github.com/alice-adventures/Alice)** is work-in-progress
+  platform that is focused on teaching Ada and SPARK by focusing on high quality
+  lessons and solutions (excelence as it is indicated in Alice's
+  documentation). One could help develop its infraestructure, testing and adding
+  lessons and tasks to it.
+- **[Exercism](https://exercism.org/)** is a learning platform where students
+  solve problems (with varying degrees of complexity) and then a tutor or
+  reviewer can give them feedback if necessary. Ada used to have an entry in the
+  website [see deprecated
+  repo](https://github.com/exercism/DEPRECATED-ada). However, due to the lack of
+  exercises and people willing to carry out the work maintaining it and helping
+  students, it is no longer available. I would be great if it could be brought
+  back. Nonetheless, Alice, see above, is trying to achive a similar goal while
+  being focused on Ada/SPARK.
+
+## Toolchains
+
+- **[Alire](https://alire.ada.dev/)** is one of the most important tools for the
+  Ada community, regardless of the experience that one may have with the
+  language. Alire is always looking for testers for new releases, feedback,
+  improvements and fixes. If you are looking for a project to help with, Alire
+  would be great!
+  - **Add new packages:** one of the easiest ways to help Alire is by adding Ada
+    projects and packages to the index and making them available for the entire
+    community.
+  - **Help with the documentation.**
+  - **Help build and test Alire in new architectures.**
+  - **Improve the [compiler
+    packaging](https://github.com/alire-project/GNAT-FSF-builds)** of Alire by
+    adding a new architecture and testing it more.
```

Comment:
There's something odd on that "testing it more": testing the new architecture more, or help to test Alire more extensively?